ComponentKit/Core/Scope/CKComponentScopeHandle.h

```diff
@@ -26,7 +27,7 @@
  handle corresponding to the current scope. Otherwise it returns nil.
  This is only meant to be called when constructing a component and as part of the implementation itself.
  */
-+ (instancetype)handleForComponent:(CKComponent *)component;
++ (instancetype)handleForComponent:(id<CKScopedComponent>)component;
```

Comment:
Github won't let me add a comment in the implementation file but shouldn't we update the signature in the implementation file as well?